### PR TITLE
User Map not Loading #7000 

### LINKED ISSUE #7000: User map not loading
**Environment**

- Live server at V5.100 (seems to work at v5.99)
- `master` branch since 06e4649

**Steps to Reproduce**

Go to `/usermap.jsp`.

**Expected Behavior**

Usermap loads.

**Actual Behavior**

image

**Reason**

Addition of an institution from "Palestine". Nothing sounds off, but "Palestine" surprisingly is not in the list of countries in `countryCodes.es6` -- what is present there is "Palestinian Territory, Occupied".

**Step to Fix**

Add `'Palestine': 'PSE'` to the country codes list.

### PR DESCRIPTION
**Pull request by GriffinHines**
Fixes #7000

Added ‘Palestine’ to the list in usermap.jsp. Also, copied all .js files from exploded-app/js into src/main/js in order to fix 404 error.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
Closing and re-opening to trigger Travis build.

File: src/main/webapp/js/countryCodes.es6 [hunk 1]
```diff
@@ -430,6 +431,7 @@ const codeToCountryName = {
     'OMN': 'Oman',
     'PAK': 'Pakistan',
     'PLW': 'Palau',
+    'PSE': 'Palestine',
     'PSE': 'Palestinian Territory, Occupied',
```
Comment (wkurniawan07) [inline src/main/webapp/js/countryCodes.es6]: There's a duplicate key here. You can remove the previous one (`Palestinian Territory, Occupied`).

**Comment by wkurniawan07** [comment]
Hi @GriffinHines, you have not resolved the change I requested.

In addition, I recommend you to run `npm run lint` and getting it to pass before pushing your changes.

**Comment by GriffinHines** [comment]
Sorry about that. I'm still getting the hang of git, it should work this time, lint checked out.

**Pull request merged**